go/database/mpt/live_trie.go

```diff
@@ -71,7 +71,7 @@ func VerifyFileLiveTrie(directory string, config MptConfig, observer Verificatio
 	if !exists {
 		return nil
 	}
-	return VerifyFileForest(directory, config, []Root{{
+	return VerifyMptState(directory, config, []Root{{
```

Comment:
Since the `LiveTrie` does not own an `MptState` it should not check for the codes. It should call `VerifyFileForest` here.